src/composables/events/ComponentSelectionEvent.js

```diff
@@ -0,0 +1,36 @@
+import { Subject } from 'rxjs';
+
+/**
+ * Represent a Event object to emit and to receive
+ * the switching of the view type ('model or 'text').
+ *
+ * @typedef {Subject} ComponentSelectionEvent
+ */
+const ComponentSelectionEvent = new Subject();
```

Comment:
Bad naming. I'm not sure but we can have an event 'ComponentEvent' and we send an event like :
next('edit', component);
and we can subscribe to this event to open drawer and set data in the form.
Feel free to propose another think, if it's better ;)

Reply:
conforms to #53